design-proposals/persist-vmi-firmware-uuid.md

```diff
@@ -0,0 +1,108 @@
+# Overview
+This proposal introduces a mechanism to make the firmware UUID of a Virtual Machine in KubeVirt universally unique.
+It ensures the UUID remains consistent across VM restarts, preserving stability and reliability.
+
+## Motivation
+* Duplicate UUIDs cause burden with third-party automation frameworks such as Gitops tools, making it difficult to maintain unique VM UUID. 
+  Such tools have to manually assign unique UUIDs to VMs, thus also manage them externally. (issue)
+* The non-persistent nature of UUIDs may trigger redundant cloud-init setup upon VM restore. (issue)
+* The non-persistent nature of UUIDs may trigger redundant license activation events during VM lifetime i.e. Windows key activation re-occur when power off and power on the VM.
+
+## Goals
+* Improve the uniqueness of the VMI firmware UUID to become independent of the VM name and namespace.
+* Maintain the UUID persistence over the VM lifecycle.
+* Maintain backward compatibility
+
+## Non Goals
+* Maintain UUID persistence with VM backup/restore.
+* Change UUID of currently existing VMs
+
+## Definition of Users
+VM owners: who require consistent firmware UUIDs for their applications.
+cluster-admin: to ensure VMs have universally unique firmware IDs
+
+## User Stories
+### Supported cases:
+* Creating and Starting a New VM
```

Comment:
Would you elaborate the story? I think it is important to state that defining a VM called X in a namespace called Y in two different clusters must created two different UUIDs